webui/react/src/components/kit/LineChart.tsx

```diff
@@ -364,10 +364,7 @@ export const ChartGrid: React.FC<GroupProps> = React.memo(
                 <FixedSizeGrid
                   columnCount={columnCount}
                   columnWidth={Math.floor(width / columnCount)}
-                  height={Math.min(
-                    height - 40,
-                    (chartsProps.length > columnCount ? 2.1 : 1.05) * 480,
```

Comment:
i dont know the context of this line so pass @johnkim-det as the code author to review the code

Reply:
Actually I have the latest blame from wrapping this code in a Spinner but I didn't actually write this. Looks like it was there back in the first version of `LineChart.tsx` by @trentwatt and @mapmeld. Don't really think I need to have written it to be able to review this current PR though, so I'll go ahead and take a look.